src/components/FieldMultiSelect/index.tsx

```diff
@@ -1,24 +1,32 @@
-import { useEffect, useState } from 'react';
+import { ReactNode, useEffect, useState } from 'react';
 
 import { FieldProps, useField } from '@formiz/core';
 import { useTranslation } from 'react-i18next';
-import { GroupBase } from 'react-select';
+import { GroupBase, MultiValue } from 'react-select';
 
-import { FieldSelectProps } from '@/components/FieldSelect';
-import { FormGroup } from '@/components/FormGroup';
-import { Select } from '@/components/Select';
+import { FormGroup, FormGroupProps } from '@/components/FormGroup';
+import { Select, SelectProps } from '@/components/Select';
+
+type MinimumOption = { value: unknown; label: ReactNode };
```

Comment:
I'm not a big fan of the name; maybe something like this will better fit the use case ?

```suggestion
type BaseOption = { value: unknown; label: ReactNode };
```